libs/api/domains/education/src/lib/graphql/main.resolver.ts

```diff
@@ -20,4 +22,17 @@ export class MainResolver {
   educationTeachingLicense(@CurrentUser() user: User) {
     return this.educationService.getTeachingLicenses(user.nationalId)
   }
+
+  @Mutation(() => SendTeachingLicense, { nullable: true })
+  educationSendTeachingLicense(
```

Comment:
`sendEducationTeachingLicense`
ref: https://github.com/island-is/island.is/blob/main/handbook/technical-overview/api-design-guide/graphql-naming-conventions.md#example